spec/src/main/asciidoc/clientexamples.asciidoc

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright (c) 2017 Contributors to the Eclipse Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[clientexamples]]
+== MicroProfile Rest Client Definition Examples
+
+MicroProfile TypeSafe Rest Clients are defined as Java interfaces.
+
+=== Sample Definitions
+
+[source, java]
+----
+public interface MyServiceClient {
+    @GET
+    @Path("/greet")
+    Response greet();
```

Comment:
I think we should say that this is a fallback, but that the "real" functionality is rather the one shown below